README.md

```diff
@@ -26,6 +26,9 @@ docker run \
   -e POSTGRES_USER="root" \
   -e POSTGRES_HOST_AUTH_METHOD="trust" \
   postgres:latest
+```
+```bash
+docker run --restart always --name postgres -v /usr/local/opt/postgres:/var/lib/postgresql/data -p 5432:5432 -d -e POSTGRES_PASSWORD="" -e POSTGRES_USER="root" -e POSTGRES_HOST_AUTH_METHOD="trust" postgres:latest
```

Comment:
What is that for? We should probably revert this.

Reply:
Multi-line command with a backslash don't work on windows. So i added the command with one line... I'll remove it :)